### PR TITLE
Freshchat API Readme Update

### PR DESCRIPTION
**Pull request by arjunpaliath**
Updated the Freshchat-api readme to include instructions on transcript generation.

### CODE REVIEW

File: packages/freshchat-api/README.md [hunk 1]
```diff
@@ -32,6 +40,23 @@ freshchat.conversationStatusUpdate('<conversation-id>', 'resolved');
...
 ```
 
+### Generate chat Transcript
+```javascript
 
-
-
+var app_url = 'https://domain.freshchat.com';
+var app_alias = '<App ID>'; //Available from Settings--> Mobile SDKs
+var conversationID = '<Freshchat Conversation UUID>';
```
Comment (arunrajkumar235) [inline packages/freshchat-api/README.md:48]: We generally use camelCase in all JS code. 
Please use `appUrl`, `appAlias`, `conversationId` for the sake of convention.

File: packages/freshchat-api/README.md [hunk 2]
```diff
@@ -32,6 +40,23 @@ freshchat.conversationStatusUpdate('<conversation-id>', 'resolved');
...
+
+freshchat.getConversationTranscript(app_url, app_alias, conversationID, {
+	//The below value can either be 'text' or 'html'
+	output: 'html', 
+	//Below is a Flag to include the conversation link in the generate transcript.
```
Comment (arunrajkumar235) [inline packages/freshchat-api/README.md:53]: ...in the generate**d** transcript.

File: packages/freshchat-api/README.md [hunk 3]
```diff
@@ -32,6 +40,23 @@ freshchat.conversationStatusUpdate('<conversation-id>', 'resolved');
...
+	isIncludeFreshchatLink: true, 
+	//Below is a Flag to generate transcript for the entire conversation or every interaction. (Create -> Resolve, Reopen -> Resolve)
```
Comment (arunrajkumar235) [inline packages/freshchat-api/README.md:55]: Create -> Latest message 
Reopen -> Latest message

This would more accurately depict what it does.

**Review by arunrajkumar235 — COMMENTED** [review]
Added a few comments.

**Comment by arjunpaliath** [comment]
@arunrajkumar235 I have made the changes as per your review. 
Please check and approve.

**Review by arunrajkumar235 — APPROVED** [review]

**Pull request merged**